src/streaming/controllers/BufferController.js

```diff
@@ -485,21 +485,17 @@ function BufferController(config) {
     function pruneBuffer() {
         if (type === 'fragmentedText') return;
 
-        var bufferToPrune = 0;
+        var start = buffer.buffered.length ? buffer.buffered.start(0) : 0;
```

Comment:
correct me if I am wrong: but I think we do not need a start variable. removing the start variable completely from this function should result in the same behavior?